src/tasks/onetime/DeliveryTask.py

```
@@ -61,11 +61,17 @@ class DeliveryTask(AccountMixin, ZipLineMixin, MapMixin):
     CFG_ONLY_DELIVER = "仅送货"
     CFG_TUTORIAL = "教程"
     CFG_DELIVERY_AREA = "地区切换"
-    CFG_TO_DELIVERY_POINT = "通向送货点"
+    CFG_TO_DELIVERY_POINT = "通向武陵城送货点"
     CFG_FULL_CYCLE_LOCATION = "完整循环测试区域"
     TUTORIAL_LINK = "https://www.bilibili.com/video/BV1LLc7zFEF9"
     TUTORIAL_TIPS = "游戏内开启全屏模式时请确保游戏内分辨率与你的屏幕分辨率一致"
 
+    # 滑索配置键迁移：旧键 → 新键
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)